content-src/experiments/voice-fill.yaml

```diff
@@ -0,0 +1,82 @@
+id: 14
+title: 'Voice Fill'
+slug: voice-fill
+platforms: ['addon']
+thumbnail: /static/images/experiments/voice-fill/experiments_experiment/thumbnail.png
+description: >
+  Speech recognition comes to Firefox. Voice fill lets you speak to your favorite search engines.
+introduction: >
```

Comment:
Echoing @flodolo's comment from below: "Any chance to split this in two strings and drop the markup?"